### DISCUSSION TITLE
Missing Filters

### DISCUSSION THREAD

**Discussion by Elitesparkle** [Heroes-Profile/heroesprofile#917] [Ideas]
The rewrite version is missing filters in various pages:

- there is no Season filter in the Hero Stats page;
- there are no Season and Hero filter in the Matchups page;
- there is no Season filter in the Roles page;
- there are no Season and Hero filter in the Maps page;
- there is no Season filter in the MMR page;
- there is no Party Size filter in the Match History page.

**Comment by Zemill** [comment]
these should all be added now

**Comment by Elitesparkle** [comment]
Thank you! I didn't test all of them but they are all showing.

When using the Season filter in the Match History page, I get no results.

**Reply by Zemill** [reply]
Hmm works for me

**Comment by Zemill** [comment]
I just tested this using your account and it works, can you double check for me?  If you find it not working, let me know what filters, if any others, you are using

**Reply by Elitesparkle** [reply]
The bug only happens when setting the Group Size filter to Solo and using the Season filter at the same time.

**Reply by Zemill** [reply]
Found why this was occurring.  Should be fixed shortly
